server/service/microsoft_mdm.go

```diff
@@ -613,6 +614,14 @@ func NewCertStoreProvisioningData(enrollmentType string, identityFingerprint str
 	return certStore
 }
 
+// IsEligibleForWindowsMDMEnrollment returns true if the host can be enrolled
+// in Fleet's Windows MDM (if it was enabled).
+func IsEligibleForWindowsMDMEnrollment(host *fleet.Host, mdmInfo *fleet.HostMDM) bool {
+	return host.FleetPlatform() == "windows" &&
+		host.IsOsqueryEnrolled() &&
+		(mdmInfo == nil || (!mdmInfo.IsServer && !mdmInfo.Enrolled))
+}
```

Comment:
not a big deal: I like this approach vs having these methods live in `fleet`. What was different about this method?

Reply:
The methods that I moved to Fleet were referenced in several other modules. I would have left them where they were, but I was running into an issue where importing them from one module to another would sometimes cause circular imports. The reason I moved them to `fleet` is because it can be imported anywhere in the system without issue.

Alternatively I could make them methods on the `Host` struct that just take a `HostMDM` struct as an argument, but then it feels a little weird to me since it doesn't really belong to one of the two structs more than the other. 

Very open to suggestions on where to move it!